src/core/include/mp-units/quantity.h

```diff
@@ -189,6 +191,25 @@ class quantity {
     return (*this).force_in(U{}).numerical_value_;
   }
 
+  // conversion operators
+  template<QuantityLike Q>
```

Comment:
The formulation of these conversion operators
look like they would fail if `Q` is a reference type.
I suspect this wouldn't work: `[](const std::chrono::seconds&) {}(1 * s)`.
Should it look something more like this?
```C++
  template<typename Qq, QuantityLike Q = std::remove_cvref_t<Qq>>
```

Reply:
Right